packages/models/tests/lessons.test.ts

```diff
@@ -856,4 +856,26 @@ describe("Lessons", () => {
       expect(first(studentsCount)?.count).to.eq(1);
     });
   });
+
+  describe(nameof(lessons.update), () => {
+    it("should successfully update lesson data", async () => {
+      const tutor = await fixtures.tutor();
+      const student = await fixtures.student();
+
+      const { lesson } = await fixtures.lesson({
+        tutor: tutor.id,
+        student: student.id,
+        timing: "past",
+        canceled: false,
+      });
+
+      const startDate = dayjs.utc().add(2, "days").toISOString();
+      const updated = await lessons.update(lesson.id, {
+        start: startDate,
+      });
+
+      expect(updated.id).to.eq(lesson.id);
+      expect(updated.start).to.eq(startDate);
```

Comment:
Let's improve this test by updating more fields (not just the start time). Also verify the the `updatedAt` field is update. Also, verify the other fields didn't change.